client/components/stripeCheckoutButton.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react'
+import StripeCheckout from 'react-stripe-checkout'
+
+const StripeCheckoutButton = ({price}) => {
+  const priceForStripe = price * 100
+  const publishablekey = 'pk_test_fuzmrsq8tsZrctgTeJmHxClb00cwD5jxSX'
```

Comment:
According to the stripe API documentation, we don't have to hide this API key, so this looks good.

Reply:
yeah, it may go to .env after backend is built